docs/low-code-spark/fabrics/synapsefabric.md

```diff
@@ -0,0 +1,78 @@
+:::
+
+A properly configured Azure Synapse Analytics environment is required before configuring a Synapse Fabric on Prophecy. Prophecy configurations include the following:
+
+1. A Synapse workspace with proper security configured. If you don't have an existing workspace, you may deploy one from the [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft.synapse?tab=overview).
+   <img src={require('./img/workspaceiam.png').default} alt="Synapse security" width="75%" />
+2. Deploy a Synapse Spark Pool if one doesn't already exist.
+   <img src={require('./img/sparkpool.png').default} alt="Synapse pool" width="75%" />
+3. Make sure the Application is registered.
+   <img src={require('./img/add_app.png').default} alt="Synapse security" width="75%" />
+4. Create a secret for your Application.
+   <img src={require('./img/secret.png').default} alt="Synapse security" width="75%" />
+5. Make a note of your secret. We need this information for your Fabric.
+   <img src={require('./img/secret2.png').default} alt="Synapse security" width="75%" />
+6. Configure Application, Workspace, Storage roles to ensure proper access.
+
+Find more information on assigning roles from [Azure's documentation](https://learn.microsoft.com/en-us/azure/synapse-analytics/security/how-to-set-up-access-control).
+
+### Configure connectivity between Synapse and Prophecy
+
+Please note that the Prophecy public IP is `3.133.35.237`.
+
+Navigate to Prophecy's UI and click on **Create Fabric**. The Fabric will establish a connection with your Synapse environment and utilize it as the execution engine for your Pipelines.
+
+**Choose Synapse** as your **Provider**.
+<img src={require('./img/provider.png').default} alt="Synapse connect" width="75%" />
+
+**Copy and paste** your **Application Client ID**, **Secret Value** and **Tenant ID** from the App Registration page.
+<img src={require('./img/appconfig.png').default} alt="Synapse connect" width="75%" />
+
+**Copy and paste** your Synapse **Resource Group Name** and **Subscription ID** from your Synapse workspace.
+<img src={require('./img/workspaceconfig.png').default} alt="Synapse connect" width="75%" />
+
+**Click** on **Fetch environments**.
+
+**Select** your Spark pool from the _Spark environment_ dropdown. All other fields should be automatically populated after selecting your Spark Pool.
+<img src={require('./img/selectsparkpool.png').default} alt="Synapse connect" width="75%" />
+
+**Click** on **Add Job Size** and configure the Job size that you would like to use for processing.
+<img src={require('./img/addjsize.png').default} alt="Synapse connect" width="75%" />
+
+Now we configure the dependencies.
+
+Under **Scala** enter **Path**
+
+`https://prophecydevpublic.blob.core.windows.net/prophecy-libs/prophecy-libs-assembly-3.3.0-ml-7.1.4.jar`
+
+Under **Python** enter **Path**
+
+`https://prophecydevpublic.blob.core.windows.net/python-prophecy-libs/prophecy_libs-1.5.9-py3.9.egg`
```

Comment:
let's mention folder paths here. We can't guarantee that `3.3.0-ml-7.1.4.jar` will continue to work for all future versions of prophecy.

Reply:
Also, this path will need to be updated once this task is done 

https://app.asana.com/0/1205057502712811/1205397769799990/f

Reply:
That's good to know. What's going to be the path?